src/Sentry.Unity/UnityEventProcessor.cs

```diff
@@ -66,6 +69,13 @@ private void PopulateDevice(Device device)
             }
         }
 
+        private void PopulateSdkIntegrations(SdkVersion sdkVersion)
+        {
+            foreach (var integrationName in _sentryOptions.SdkIntegrationNames)
```

Comment:
`SdkIntegrationNames` is of a mutable type, and if we iterate on it here while it's being changed on another thread, it'll throw.

Looking at the code here, it's only mutated in the event processor's constructor so it will be fine. But as a follow up we should take that into account (make `SdkIntegrationNames` copy on write

Reply:
Good point! Thanks